Code/Editor/EditorViewportWidget.cpp

```diff
@@ -1957,6 +1979,24 @@ void EditorViewportWidget::SetDefaultCamera()
     PostCameraSet();
 }
 
+
+//////////////////////////////////////////////////////////////////////////
+void EditorViewportWidget::SetDefaultCameraNearFar()
+{
+    auto m = m_defaultView->GetViewToClipMatrix();
+    AZ::SetPerspectiveMatrixNearFar(m, ed_editor_camera_near, ed_editor_camera_far, true);
```

Comment:
suggestion: Would be good to indicate what the bool parameter is here too to make the calling code clearer (e.g. `/*reverseZ=*/ true`)